files/zh-cn/mdn/community/contributing/translated_content/index.md

```diff
@@ -1,28 +1,27 @@
 ---
-title: MDN 的本地化
+title: MDN Web 文档的本地化
 slug: MDN/Community/Contributing/Translated_content
-original_slug: MDN/Contribute/Localize
 ---
 
 {{MDNSidebar}}
 
-2020 年 12 月 14 日开始，MDN 将运行在基于 Github 的 [Yari 平台](https://github.com/mdn/yari)上。新的平台也使 MDN 在许多方面发生了改变，这些变化基本上都对我们的工作有不同程度的改进，不过，在内容本地化的工作方面，变化可能看起来会更激进一些。许多的英语区域外的内容本地化工作被中止了，其中的内容长久未更新或活跃人数太少都是可能的原因。在将来，我们希望有更好的机制，来管理这些内容的本地化工作。
+自 2020 年 12 月 14 日起，MDN 一直运行在新的基于 Github 的 [Yari 平台](https://github.com/mdn/yari)上。这为 MDN 带来了许多积极影响，但我们需要彻底改变我们处理本地化内容的方式。因为我们有许多不再维护且过时的本地化内容，我们希望在未来能够更好地管理这些内容。
 
-除了下面列出的地区，所有区域的内容本地化工作都被冻结了，也就意味着这些内容暂时不会开放编辑。所有可接受本地化编辑的内容，都由各自对应的志愿者小组负责管理。
+除了以下语言——这些语言有专门的团队负责维护，其他所有的本地化内容都被冻结了。也就意味着，我们不再接受对这些内容的任何更新，它们是只读的。
 
 ## 活跃语言
 
-> **备注：** 如果你有意向加入某语言的内容本地化工作，请联系下面这些小组中的成员，也可 [尝试寻求帮助](/zh-CN/docs/MDN/Contribute/Getting_started#step_4_ask_for_help)。语言名称后的标识与 [Translated-content Repo](https://github.com/mdn/translated-content) 目录是对应的。
+> **备注：** 如果你有意向为某个活跃语言做出贡献，请联系下面这些小组中的成员，或[联系我们以寻求帮助](/zh-CN/docs/MDN/Community/Communication_channels)。
 
 ### 巴西葡萄牙语（pt-BR）
 
 - 讨论组：[Telegram (MDN localization in Brazilian Portuguese)](https://t.me/mdn_l10n_pt_br)
-- 目前的志愿者：[Luisa Migueres](https://github.com/lumigueres)、[Julio Sampaio](https://github.com/juliosampaio)、[Josiel Rocha](https://github.com/josielrocha)、[Clóvis Lima Júnior](https://github.com/clovislima)
+- 目前的志愿者：[Nathália Pissuti](https://github.com/nathipg)、[Josiel Rocha](https://github.com/josielrocha)、[Clóvis Lima Júnior](https://github.com/clovislima)
 
 ### 汉语（zh-CN、zh-TW）
 
 - 讨论组：[Telegram (MozTW L10n channel)](https://moztw.org/tg)、[Telegram (Mozilla China channel)](https://t.me/mozilla_china)
```

Comment:
```suggestion
- 讨论组：[Telegram (MozTW L10n channel)](https://moztw.org/community/telegram/)、[Telegram (Mozilla China channel)](https://t.me/mozilla_china)
```

Reply:
The en-US document just using the short-url. This may not necessary to update it.

Reply:
https://github.com/yin1999/content/blob/0aa8ef68a2c32b45b6636510309a48f6d24ad673/files/en-us/mdn/community/contributing/translated_content/index.md?plain=1#L24

seem interesting, the source says that they are using the long one

Reply:
See, the redirection has been resolved in mdn/content#15582